main.tf

```diff
@@ -230,16 +230,22 @@ module "adminer" {
  * Create new ecs service
  */
 module "ecs" {
-  source             = "github.com/sil-org/terraform-modules//aws/ecs/service-only?ref=8.13.3"
+  source  = "sil-org/ecs-service/aws"
+  version = "~> 0.3.0"
```

Comment:
Interesting version change 🤔

Reply:
Version numbering started over. From terraform-modules monorepo to individual repos, it made sense to start over at zero.